app/javascript/App/bundles/common/components/LoadingIndicator.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react'
+
+const loadingImg = `<svg version="1.1" id="loader-1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
```

Comment:
I think it may be better to keep this as a separate `.svg` file that is imported in